utils/path.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"os/user"
+	"path"
+	"runtime"
+)
+
+// package-level variables for mocking in tests
+var (
+	userCurrent = user.Current
+	getGOOS     = runtime.GOOS
+)
+
+// GetDefaultCliConfigPath returns the default path for the ArduinoCLI configuration file.
+func GetDefaultCliConfigPath() string {
+	if user, _ := userCurrent(); user != nil {
+		return path.Join(user.HomeDir, func() string {
+			switch getGOOS {
+			case "darwin":
+				return "Library/Arduino15"
+			default:
+				return ".arduino15"
```

Comment:
The default path on Windows is in `%LOCALAPPDATA%/Arduino15` (i.e., `C:/Users/<username>/AppData/Local/Arduino15/arduino-cli.yaml`).

Reply:
I added a case covering Windows. It does not support Microsoft Store installations according to the ArduinoCLI documentation.